File: maui-sample/Models/ReceiverInfo.cs
namespace MauiSample.Models
{
  public class ReceiverInfo
  {
    public bool IsReceiverConfigured { get; set; }
    public string? BluetoothName { get; set; }
    public string? BluetoothAddress { get; set; }
    public string? ReceiverBrand { get; set; }
    public string? ReceiverModel { get; set; }
    public string? ReceiverSerialNumber { get; set; }
    public bool IsConnected { get; set; }
    public bool IsSigninRequired { get; set; }
    public bool IsSignedIn { get; set; }
  }
}

File: android-native-sample/app/src/main/java/com/example/kotlin_sample/models/ReceiverInfo.kt
package com.example.kotlin_sample.models

import kotlinx.serialization.Serializable

@Serializable
data class ReceiverInfo(
  val isReceiverConfigured: Boolean = false,
  val bluetoothName: String? = null,
  val bluetoothAddress: String? = null,
  val receiverBrand: String? = null,
  val receiverModel: String? = null,
  val receiverSerialNumber: String? = null,
  val isConnected: Boolean = false,
  val isSigninRequired: Boolean = false,
  val isSignedIn: Boolean = false,
)
